jni/utilities/samplemanager.cpp

```diff
@@ -29,24 +29,34 @@ namespace SampleManagerSamples
 
 /* public methods */
 
-void SampleManager::setSample( std::string aKey, AudioBuffer* aBuffer )
+void SampleManager::setSample( std::string aIdentifier, AudioBuffer* aBuffer )
 {
     cachedSample sample = { aBuffer->bufferSize, aBuffer };
 
     // Assignment using member function insert() and STL pair
-    SampleManagerSamples::_sampleMap.insert( std::pair<std::string, cachedSample>( aKey, sample ));
+    SampleManagerSamples::_sampleMap.insert( std::pair<std::string, cachedSample>( aIdentifier, sample ));
 }
 
 AudioBuffer* SampleManager::getSample( std::string aIdentifier )
 {
+    if ( !hasSample( aIdentifier ))
+        return 0;
+
```

Comment:
Previously these getter methods assumed the sample existed, that is a little dangerous.